app/views/shared/_checkbox_group.html.haml

```diff
@@ -0,0 +1,12 @@
+- if search
+  .search-input
+    %input.govuk-input.search-input__input{ placeholder: "Search" }
+    %button{type: "submit"}
+      %i.fa.fa-search
+.govuk-form-group
+  .govuk-checkboxes.govuk-checkboxes--small
+    = form_for '', url: 'url', html: { method: 'post' } do |f|
+      = f.govuk_collection_check_boxes :content,
+        options,
+        :first,
+        :first
```

Comment:
If we were using VacanciesHelper's `working_pattern_options`, for example, then this would need to be
```ruby
options,
:second,
:first
```
where `:second` gets the computer-readable name and `:first` gets the human-readable name.

Is there a way to allow this type of options to be used by the checkbox group?

I *think* if instead of `:second` you use `:last`, then it should work for both types, because in any element of a one-dimensional array its last and first elements are the same element, could be wrong about how it works though.

Alternatively, if we can't easily abstract this further, you could add a comment somewhere, e.g. top of `app/views/shared/_filter_group.html.haml`, to say what format the list of options needs to take (a one-dimensional array). This is less preferred (by me) because it means we can't separate human-readable and computer-readable names.

Reply:
Correct me if I am misunderstanding something!

Reply:
no you are not misunderstanding. im not sure about those params cause i was struggling to find docs that werent just examples and tbh i used the bare minimum for the example data i was using from the vacancy presenter. although working patterns and subject data works fine without change. i think this is something to standardise when the components are being used. but in the scope of this PR i am unsure. i think its best to document this (with a few other bits in the PR) so ill do that.

Reply:
simplified the arguments need for the checkbox list and put a link to the proper docs page

Reply:
`:last` and `:first` makes sense to me 👍